### PR TITLE
Data streaming

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
This PR adds some replication settings and a postgres module to enable the use of data streaming...

With this change the replication is "ready" but not actually active.

### CODE REVIEW

File: bin/requirements.yml [hunk 1]
```diff
@@ -19,3 +19,6 @@
 
 - src: geerlingguy.postgresql
   version: 1.4.5
+
+- src: libre_ops.wal2json
```
Comment (sauloperez) [inline bin/requirements.yml]: Who is exactly does not understand WAL and needs JSON instead?

Comment (Matt-Yorkley) [reply bin/requirements.yml]: Everybody needs JSON! :trollface: 

Basically:
```
Postgres -> Wal2json ->  Kafka queue -> JDBC driver -> Postgres replica
```

But it also means that other things can read those database changes from the queue and parse them easily.

**Comment by Matt-Yorkley** [comment]
Looks like the travis box has multiple postgres versions installed, and it's picked the wrong one...

File: roles/dbserver/tasks/main.yml [hunk 2]
```diff
@@ -35,3 +35,8 @@
 - import_tasks: fix_template_encoding.yml
   tags: template_encoding
 
+- name: install wal2json module
+  include_role:
+    name: libre_ops.wal2json
+  vars:
+    wal2json_set_preload_libararies: false
```
Comment (sauloperez) [inline roles/dbserver/tasks/main.yml]: :warning: isn't `libararies` a typo?

File: inventory/group_vars/all.yml [hunk 3]
```diff
@@ -64,6 +64,8 @@ postgres_encoding: en_US.utf8
 # to the defaults if present. Make sure you use the "list" notation with the dashes!
 
 default_hba_entries:
+  - {type: local, database: replication, user: "{{ db_user }}", auth_method: trust}
+  - {type: host, database: replication, user: "{{ db_user }}", address: '127.0.0.1/32', auth_method: trust}
```
Comment (sauloperez) [inline inventory/group_vars/all.yml]: I don't understand these entries. There won't be any `replication` database in any of our servers. Aren't these the settings the new DB replica will have?

**Comment by Matt-Yorkley** [comment]
> @Matt-Yorkley This won't be an issue in production and/or staging servers?

The issue I was having was related to building the wal2json module from source, and was definitely only in the Travis environment. I discovered that wal2json is now available as a prebuilt package directly from the official Potsgres repo, so I switched to that instead.

File: inventory/group_vars/all.yml (hunk 3 above)
Comment (Matt-Yorkley) [reply inventory/group_vars/all.yml]: It looks weird, but this is how it needs to be written, based on what postgres expects and how the role writes entries to the file. It's not an extra database, it's basically permission to perform replication.

Docs:
> "The value `replication` specifies that the record matches if a replication connection is requested (note that replication connections do not specify any particular database)."

https://www.postgresql.org/docs/9.5/auth-pg-hba-conf.html

File: roles/dbserver/tasks/main.yml (hunk 2 above)
Comment (Matt-Yorkley) [reply roles/dbserver/tasks/main.yml]: That's how it's spelled in the role too. :see_no_evil: I'll fix it.

Comment (Matt-Yorkley) [reply roles/dbserver/tasks/main.yml]: Done: https://github.com/openfoodfoundation/ofn-install/pull/468/commits/2a16f81028f58590833423c1c0f35651977380cc

**Pull request merged**